.github/workflows/main.yml

```diff
@@ -1547,12 +1632,6 @@ jobs:
           java-version: ${{ matrix.java }} 
           distribution: 'zulu'
 
-      - uses: actions/cache@v3
-        with:
-          path: ~/Library/Caches/Homebrew
-          key: ${{ runner.os }}-homebrew
-          restore-keys: ${{ runner.os }}-homebrew
-
```

Comment:
the mac job cached about 3.4 GB of files without actually having to do that. It installed ant with brew (few lines later), but ant is already on the system (even in the same version).

```
Run brew install ant
Warning: ant 1.10.12 is already installed and up-to-date.
To reinstall 1.10.12, run:
  brew reinstall ant
```


Maybe I am missing something obvious here (the ant tests don't use brew for anything, right?), but I think we can simply remove caching on the mac :)